package.json

```diff
@@ -48,6 +49,8 @@
     "@angular/compiler-cli": "~13.3.0",
     "@types/jasmine": "~3.10.0",
     "@types/node": "^12.11.1",
+    "cross-env": "7.0.3",
+    "http-server": "^14.1.1",
```

Comment:
you can remove these deps, I only added them to help with testing the issue (I had used them in the package.json scripts)

Reply:
i thought it was helpful for testing prod builds locally so i kept it in